src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java

```diff
@@ -23,4 +29,38 @@ private Et3ResponseHelper() {
     public static String formatClaimantNameForHtml(CaseData caseData) {
         return String.format(CLAIMANT_NAME_TABLE, caseData.getClaimant());
     }
+
+    /**
+     * Validates that the employment start date is in the past and not after 
+     * the employment end date if both dates are provided.
+     * @param caseData data for the current case
+     * @return List of validation errors encountered
+     */
+    public static List<String> validateEmploymentDates(CaseData caseData) {
+        ArrayList<String> errors = new ArrayList<>();
+        String startDateStr = caseData.getEt3ResponseEmploymentStartDate();
+        String endDateStr = caseData.getEt3ResponseEmploymentEndDate();
+
+        if (isNullOrEmpty(startDateStr)) {
+            return errors;
+        }
+
+        LocalDate startDate = LocalDate.parse(startDateStr);
+
+        if (startDate.isAfter(LocalDate.now())) {
+            errors.add("Start date must be in the past");
```

Comment:
Can error messages be constant strings as you defined in tests? They can be defined here and test class can reuse them